CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.28)
+project(raptorxx VERSION 0.0.1)
```

Comment:
Set this version to be consistent with the version (`v1.0.0`) in vcpkg.json. Otherwise the build seems to fail on unix systems.

Reply:
I have found out how to solve this correctly. 
I will implement this